src/components/MessageList/CustomNotification.tsx

```diff
@@ -1,19 +1,25 @@
 import React, { PropsWithChildren } from 'react';
+import clsx from 'clsx';
 
 export type CustomNotificationProps = {
   type: string;
   active?: boolean;
+  className?: string;
 };
 
 const UnMemoizedCustomNotification = (props: PropsWithChildren<CustomNotificationProps>) => {
-  const { active, children, type } = props;
+  const { active, children, className = '', type } = props;
 
   if (!active) return null;
 
   return (
     <div
       aria-live='polite'
-      className={`str-chat__custom-notification notification-${type} str-chat__notification`}
+      className={clsx(
+        `str-chat__custom-notification notification-${type}`,
+        `str-chat__notification`,
+        { [className]: className },
```

Comment:
```suggestion
        className,
```
This is sufficient. :)